ios/Sources/HomeFeature/HomeFeature.swift

```diff
@@ -1,11 +1,46 @@
+import Component
 import ComposableArchitecture
 
+// dummy feed model
+public struct FeedItem: Equatable, Identifiable {
+    public var id: String
+    public var imageURLString: String
+    public var link: String
+    public var media: TagType
+    public var publishedAt: Date
+    public var summary: String
+    public var title: String
+
+    public init(id: String, imageURLString: String, link: String, media: TagType, publishedAt: Date, summary: String, title: String) {
```

Comment:
maybe adding newline for each param might be better to read?

Reply:
I think it is not needed because this is dummy(will be removed soon).